hawc/apps/materialized/admin.py

```diff
@@ -0,0 +1,49 @@
+from django.contrib import admin
+
+from ..common.admin import ReadOnlyAdmin
+from . import models
+
+
+class IsOverrideFilter(admin.SimpleListFilter):
```

Comment:
can this a more generic class, maybe moved to common, call `IsNullFilter` Or maybe a generic one with something like this:

```python
class IsOverrideFilter(IsNullFilter):
    qs_filter = "object_id"
```

Reply:
I made a generic `YesNoFilter` that you can set the inclusion criteria on as a Q object. You kind of have to go out of your way to make a Q object when you inherit from this class, but I think its more flexible and can be used more widely on our other admin pages this way.

Reply:
that sounds great, using Q object is a great idea b/c that adds a lot of flexibility